doc/src/build/zk_login.md

```diff
@@ -505,21 +513,25 @@ In addition, each wallet or application maintains its own `user_salt`, so loggin
 
 See more on address [definition](#address-definition).
 
+## Can I have multiple addresses with the same OAuth provider?
+
+Yes, this is possible by using a different wallet provider or different `user_salt` for each account. This is useful for separating funds between different accounts.
+
 ## Is a zkLogin Wallet custodial?
 
 A zkLogin wallet is a non-custodial or unhosted wallet.
 
 A custodial or hosted wallet is where a third party (the custodian) controls the private keys on behalf of a wallet user. No such third-party exists for zkLogin wallets.
 
-Instead, a zkLogin wallet can be viewed as a 2-out-of-2 Multisig where the two credentials are the user's OAuth credentials (maintained by the user) and the salt. In other words, neither the OAuth provider, the ZK proving service or the salt service provider (if not maintained by the user) is a custodian.
+Instead, a zkLogin wallet can be viewed as a 2-out-of-2 Multisig where the two credentials are the user's OAuth credentials (maintained by the user) and the salt. In other words, neither the OAuth provider, the wallet vendor, the ZK proving service or the salt service provider is a custodian.
 
 ## Generating a ZK proof is expensive, is a new proof required to be generated for every transaction?
 
 No. Proof generation is only required when ephemeral KeyPair expires. Since the nonce is defined by the ephemeral public key (`eph_pk`) and expiry (`max_epoch`), the ZK proof is valid until what the expiry is committed to nonce in the JWT token. The ZK proof can be cached and the same ephemeral key can be used to sign transactions till it expires.
 
 ## Does zkLogin work on mobile?
 
-zkLogin works with any wallet that is whitelisted and has completed integration with it, including mobile wallets.
+zkLogin is a a Sui native primitive and not a privilege of a particular wallet provider, and thus it can be integrated to any wallet provider, including mobile wallets.
```

Comment:
```suggestion
zkLogin is a Sui native primitive and not a feature of a particular application or wallet. It can be used by any Sui developer, application, or wallet.
```